### ISSUE TITLE
Give user a hint where to find the running ui

### ISSUE THREAD

**Issue by themalkolm** [iverberk/nomad-ui#17]
```
$ NOMAD_ADDR="localhost" ./nomad-ui-linux
2016/08/15 10:47:58 Starting server...
```

Whenever I start it says it is "starting" but leaves me wondering if:

1. Did it actually start?
2. Where exactly it is running?

What if we write something like this instead to not confuse user:

```
$ NOMAD_ADDR="localhost" ./nomad-ui-linux
2016/08/15 10:47:58 Starting server...
2016/08/15 10:47:58 Started server at localhost:3000
```

This way I see that it is started and obvious where to find it. What do you think?

**Comment by iverberk** [comment]
Very valid points and good suggestion! Care to create a PR for this? :-)

**Comment by iverberk** [comment]
Fixed by #27
